cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -99,8 +99,14 @@ if("@MSVC@")
   set(bootstrap_cmd "bootstrap.bat")
 elseif("@MINGW@")
   # Test scenario: MinGW installed in system, Visual Studio - not
-  set(install_cmd "b2")
-  set(bootstrap_cmd "bootstrap.bat" "gcc")
+  if("${CMAKE_CROSSCOMPILING}")
+    # cross compile from "linux" to "windows" using mingw
+    set(install_cmd "./b2")
```

Comment:
Yes we need `gcc`. We need this if there is no Visual Studio but only MinGW installed:
* https://github.com/ruslo/hunter/pull/753